README-fr.md

```diff
@@ -119,16 +119,16 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
 * [Load balancer](#load-balancer)
     * [Active-passive](#active-passive)
     * [Active-active](#active-active)
-    * [Layer 4 load balancing](#layer-4-load-balancing)
-    * [Layer 7 load balancing](#layer-7-load-balancing)
+    * [Répartition de charge couche 4](#layer-4-load-balancing)
+    * [Couche 7 load balancing](#layer-7-load-balancing)
     * [Horizontal scaling](#horizontal-scaling)
 * [Reverse proxy (web server)](#reverse-proxy-web-server)
     * [Load balancer vs reverse proxy](#load-balancer-vs-reverse-proxy)
-* [Application layer](#application-layer)
+* [Couche Applicative](#application-layer)

```

Comment:
```suggestion
* [Couche application](#application-layer)
```
"application" seems to be the right word(https://fr.wikipedia.org/wiki/Couche_application) as you suggested before, I'm just removing the capital letter